node_package/src/ReactOnRails.js

```diff
@@ -0,0 +1,35 @@
+import clientStartup from './clientStartup';
+import handleError from './handleError';
+import ComponentStore from './ComponentStore';
+import serverRenderReactComponent from './serverRenderReactComponent';
+import buildConsoleReplay from './buildConsoleReplay';
+
+const context = ((typeof window !== 'undefined') && window) ||
+  ((typeof global !== 'undefined') && global) || this;
```

Comment:
could we extract a variable or two just for readability; `context` is a boolean?